pyicloud/services/calendar.py

```diff
@@ -41,7 +41,12 @@ def refresh_client(self, from_dt=None, to_dt=None):
         today = datetime.today()
         first_day, last_day = monthrange(today.year, today.month)
```

Comment:
`monthrange` is documented as: 
>Returns weekday of first day of the month and number of days in month, for the specified year and month.
So this line should be:

 `weekday, num_days = monthrange(today.year, today.month)`